### ISSUE TITLE
setup is broken

### ISSUE THREAD

**Issue by flowolf** [wuttem/cattledb#4]
current pip breaks the setup of cattledb 😢

**Comment by wuttem** [comment]
What Version of pip and cattledb are you using ?

**Comment by flowolf** [comment]
```Collecting cattledb>=0.6.0
  Downloading cattledb-0.6.1.tar.gz (650 kB)
    ERROR: Command errored out with exit status 1:
     command: /usr/bin/python3 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-57clplom/cattledb/setup.py'"'"'; __file__='"'"'/tmp/pip-install-57clplom/cattledb/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-5g2dmmo9
         cwd: /tmp/pip-install-57clplom/cattledb/
    Complete output (7 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-57clplom/cattledb/setup.py", line 42, in <module>
        reqs = [str(ir.req) for ir in install_reqs]
      File "/tmp/pip-install-57clplom/cattledb/setup.py", line 42, in <listcomp>
        reqs = [str(ir.req) for ir in install_reqs]
    AttributeError: 'ParsedRequirement' object has no attribute 'req'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
Skipping cache upload for failed step```

**Comment by wuttem** [comment]
Fixed in Version 0.6.2
setup.py is now compatible with pip>=20.1
